### PR TITLE
feat: Add Python plugin. Fixes #7913

### LINKED ISSUE #7913: Python plugin
A Python plugin that allows you to execute basic Python scripts.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by alexec**
Signed-off-by: Alex Collins <alex_collins@intuit.com>

Fixes #7913

### CODE REVIEW

File: plugins/executor/python/python-executor-plugin-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# This is an auto-generated file. DO NOT EDIT
```
Comment (terrytangyuan) [inline plugins/executor/python/python-executor-plugin-configmap.yaml:1]: Perhaps I missed this somewhere but have we documented how to auto-generate this?

File: plugins/executor/python/server.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import json
```
Comment (terrytangyuan) [inline plugins/executor/python/server.py:1]: Should we expect plugin authors to write some tests?

Comment (alexec) [reply plugins/executor/python/server.py:1]: good question - we should discus this - I think we should expect quality - band that may need tests - what kind of tests?

File: plugins/executor/python/python-executor-plugin-configmap.yaml (hunk 1 above)
Comment (alexec) [reply plugins/executor/python/python-executor-plugin-configmap.yaml:1]: `argo executor-plugin build .`

**Pull request closed without merging**